src/coreclr/jit/gentree.cpp

```diff
@@ -3948,26 +4015,6 @@ unsigned Compiler::gtSetEvalOrder(GenTree* tree)
                     costSz = 2 * 2;
                     break;
 
-#if defined(FEATURE_HW_INTRINSICS) && defined(TARGET_XARCH)
-                case GT_HWINTRINSIC:
```

Comment:
@SingleAccretion could you please explain why this was removed?

Reply:
Hmm, this code was just moved to `gtSetMultiOpOrder`.

Reply:
@SingleAccretion ah, I see, thanks